src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -27,10 +27,9 @@ namespace Tizen.NUI
     /// <summary>
     /// WebView
```

Comment:
Please add more description for this class

Reply:
Done.